CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+## 3.26.0
+- [SQLite 3.26.0](http://sqlite.org/releaselog/3_26_0.html)
+- Update `org.apache.httpcomponents:httpclient` dependency to 4.5.6
+- Update `com.android.tools.build:gradle` dependency to 3.2.1
```

Comment:
Generally it's not needed to mention changes that don't effect the user of the library, it's fine for now but may amend later